### PR TITLE
Fix wallpaper_command commandline building

### PR DESCRIPTION
**Pull request by markhellewell**
Also, refresh wallpaper list when button is pressed

### CODE REVIEW

File: libqtile/widget/wallpaper.py [hunk 1]
```diff
@@ -81,8 +81,9 @@ def set_wallpaper(self):
         else:
             self.text = self.label
         if self.wallpaper_command:
-            self.wallpaper_command.append(cur_image)
-            subprocess.call(self.wallpaper_command)
+            wallpaper_command = self.wallpaper_command.copy()
+            wallpaper_command.append(cur_image)
+            subprocess.call(wallpaper_command)
             self.wallpaper_command.pop()
```
Comment (ramnes) [inline libqtile/widget/wallpaper.py]: Shouldn't we drop this `pop()`, then?

Comment (markhellewell) [reply libqtile/widget/wallpaper.py]: Sorry, you’re right, have updated

**Comment by ramnes** [comment]
What does it fix, exactly?

**Comment by markhellewell** [comment]
Actually, you know what? I fixed a problem that only exists in the 0.10.7 version which ships with ubuntu 18.04. In that version the commandline grows and grows because the list never gets popped :(

Disregard this PR. I’ll get set up with a way to run a more recent version of qtile!

**Pull request closed without merging**